### PR TITLE
fixed platform citations and comments added. Updated function calls to be consistent with ORBIT 1.06. 

### PR DESCRIPTION
**Pull request by nRiccobo**
Added ORBIT citation and references. Commented each function, citing reference section/equations used or repurposed from this analysis. Also included additional resource that cites default values.

### CODE REVIEW

**Comment by nRiccobo** [comment]
In a recent release of ORBIT 1.06, the _design_phases changed from a list to a tuple data type. Therefore, previous calls that use _design_phase.append() no longer apply. 

I updated any .append() calls in the test section of fixed_platform.py to use the register_design_phase() and register_install_phase() modules. Now, the new phase argument gets added and treated as any phase in ORBIT.

File: hopp/offshore/fixed_platform.py [hunk 1]
```diff
@@ -30,8 +36,9 @@
 """
 ''' 
 Notes:
-    Thank you Jake Nunemaker's oswh2 repository!!!
-    pile_cost=0 $US/tonne for monopile construction. Not a bug, this # is consistent with the rest of ORBIT
+    - Thank you Jake Nunemaker's oswh2 repository!!!
```
Comment (jaredthomas68) [inline hopp/offshore/fixed_platform.py:39]: Can you point me to this repository? It sounds helpful.

File: hopp/offshore/fixed_platform.py [hunk 2]
```diff
@@ -215,20 +226,20 @@ def detailed_output(self):
...
     topside_design_cost     =   design_cost
 
-    '''Topside Cost & Mass
+    '''Topside Cost & Mass (repurposed eq. 2.26 from [2])
     Topside Mass is the required Mass the platform will hold
     Topside Cost is a function of topside mass, fab cost and design cost'''
     topside_cost   =   topside_mass   *topside_fab_cost_rate  +topside_design_cost
 
-    '''Substructure
+    '''Substructure (repurposed eq. 2.31-2.33)
```
Comment (jaredthomas68) [inline hopp/offshore/fixed_platform.py:242]: From what reference?

**Review by jaredthomas68 — APPROVED** [review]

File: hopp/offshore/fixed_platform.py (hunk 1 above)
Comment (nRiccobo) [reply hopp/offshore/fixed_platform.py:39]: It was just a private repository that Jake shared with me from a previous offshore-h2 analysis. We'd have to reach out to him for access.

File: hopp/offshore/fixed_platform.py (hunk 2 above)
Comment (nRiccobo) [reply hopp/offshore/fixed_platform.py:242]: It should be from [2]. I'll update the comment.

**Pull request merged**